app/controllers/users_controller.rb

```diff
@@ -1,3 +1,39 @@
 class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
 
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      @user.send_confirmation_email
+      flash[:success] = "Please confirm your email address to continue"
```

Comment:
Can we add a POST request to also create the user in our backend once the email confirmation is complete?